redhat-developer/app-services-cli
# fix: enable self updating CLI

Add ability for CLI to self update itself. This requires more testing with actual release builds and general improvements


This PR will only work for latest release with manual darwin binary.
We need fix upstream: https://github.com/rhysd/go-github-selfupdate/pull/48

## Verification

0. change version in the makefile to 0.1.0
1. Run make binary
2. Run `rhoas kafka list -v`
3. Press y :D
4. CLI should be updated.
5. Execute command again (you should not see any message as CLI have been updated to latest)


Putting binary into system folder would fail and will require sudo to update
